greenberry/gbtest/test.py

```diff
@@ -21,5 +21,15 @@ def test_null(self):
         self.assertTrue(error)
 
 
+    def test_eval_add(self):
```

Comment:
can you add parameter instead of hardcoding the value? that is, test_eval_add(self, x) so that we can do actual test in another file? thanks

Reply:
sure!